checker/src/javax/annotation/concurrent/GuardedBy.java

```diff
@@ -41,19 +44,23 @@
 // to the Lock Checker's @GuardedBy annotation) or on a method (in which case
 // it is a declaration annotation corresponding to the Lock Checker's @Holding
 // annotation).
+// It is preferred to use these Checker Framework annotations instead:
+//  org.checkerframework.checker.lock.qual.GuardedBy
+//  org.checkerframework.checker.lock.qual.Holding
```

Comment:
We should only support them as aliases. For that they don't need to have TYPE_USE.